examples/pysdl2.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/python2
+
+#
+# Simple SDL2 / cefpython3 example.
+#
+# Only handles mouse events but could be extended to handle others.
+#
+# Requires pysdl2 (and SDL2 library).
+#
+# Tested under Fedora Linux (x86_64).
+#
+# by Neil Munday (www.mundayweb.com)
+#
+
+import os
+import sys
+import sdl2
+import sdl2.ext
+
+from cefpython3 import cefpython as cef
+from PIL import Image
+
```

Comment:
Two blank lines required according to Pep8. See the Contributing-code.md page for details on code style requirements.